src/main/kotlin/uk/gov/dwp/dataworks/kafkatohbase/reconciliation/services/impl/TrimReconciledRecordsServiceImpl.kt

```diff
@@ -0,0 +1,51 @@
+package uk.gov.dwp.dataworks.kafkatohbase.reconciliation.services.impl
+
+import org.springframework.context.annotation.Profile
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Service
+import uk.gov.dwp.dataworks.kafkatohbase.reconciliation.configuration.ReconcilerConfiguration
+import uk.gov.dwp.dataworks.kafkatohbase.reconciliation.repositories.MetadataStoreRepository
+import uk.gov.dwp.dataworks.kafkatohbase.reconciliation.services.ReconciliationService
+import uk.gov.dwp.dataworks.kafkatohbase.reconciliation.services.TrimReconciledRecordsService
+import uk.gov.dwp.dataworks.logging.DataworksLogger
+
+@Service
+@Profile("TRIM_RECONCILED_RECORDS")
+class TrimReconciledRecordsServiceImpl(
+    private val reconcilerConfiguration: ReconcilerConfiguration,
+    private val metadataStoreRepository: MetadataStoreRepository
```

Comment:
Can we have the trim string variables passed here and not into the metadatastore

Reply:
Indeedio